ami_builder.py

```diff
@@ -22,26 +26,25 @@ def handler(event, context):
         boto3.set_stream_logger()
         logger.debug(f"Using boto3 {boto3.__version__}")
 
-    if 'packer_template_bucket_region' in event:
-        s3_url = f"https://s3.{event['packer_template_bucket_region']}.amazonaws.com"
+    s3_url = f"https://s3.{event['Apacker_template_bucket_region']}.amazonaws.com"
```

Comment:
Is that leading 'A' a typo?

Reply:
ha, not really, was testing the key error.
I was literally hovering over the merge button when you commented